CODESAMPLESLIST.md

```diff
@@ -157,13 +158,10 @@ This Page is a complete list of oneAPI Code Samples, sorted by Alphabetical orde
 |[Tutorials](https://github.com/oneapi-src/oneAPI-samples/tree/master/Libraries/oneCCL/tutorials)|['CPU', 'GPU']|Intel® oneCCL Tutorials|
```

Comment:
Insufficient sample name, this is an issue with the samples json, not the code provided here

The json needs updating to be updated with a clear title, something like **Tutorials: oneCCL Getting started** - check with Louie Tsai for recommended change

Reply:
I think that this name is good enough, it is not one sample but it should be list of jupyter notebooks.